xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h

```diff
@@ -50,6 +50,11 @@
                                                       "libXBMC_addon.h" \
                                                       "addon-instance/"
 
+#define ADDON_GLOBAL_VERSION_GENERAL                  "1.0.0"
+#define ADDON_GLOBAL_VERSION_GENERAL_MIN              "1.0.0"
+#define ADDON_GLOBAL_VERSION_GENERAL_XML_ID           "kodi.binary.global.general"
+#define ADDON_GLOBAL_VERSION_GENERAL_DEPENDS          "General.h"
```

Comment:
There coming on next requests two another global parts, one for network and one for filesystem. Then is about globals for the moment everything done.

Will keep them apart to avoid possible problems with changes.